### PR TITLE
Feature/ 312 Create a unit test for the wallet creation method in the wallet controller

### PR DESCRIPTION
**Pull request by RobertoJavaDev**
#### Checklist template for LEADERS

&#x2611; | &#x2610;

&#x2610; New code is covered with tests and coverage requirements are met.
&#x2610; Build is not failing.
&#x2610; All tests are passing.
&#x2610; All conflicts are resolved and the branch is rebased on top of the up-to-date parent branch.
&#x2610; All issue requirements are implemented.
&#x2610; At least two reviewers are assigned.
&#x2610; The Pull request is linked to the related issue.
&#x2610; Correct branch name. Convention [https://github.com/bycza-zagroda/track-expenses-app-documentation/blob/develop/githubwork/GITHUB_WORK.md]
&#x2610; Correct unit test name - suffix 'Test' in the end.
&#x2610; Correct integration test name - suffix 'IT' in the end.
&#x2610; Positive and negative test paths have been implemented.

### CODE REVIEW

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletCreateControllerTest.java [hunk 1]
```diff
@@ -48,6 +48,8 @@ class WalletCreateControllerTest {
 
     public static final String TOO_LONG_NAME_MORE_THAN_20_LETTERS = "Too long name - more than 20 letters.";
 
+    public static final String WALLET_NAME = "test";
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletCreateControllerTest.java]: Maybe "
```suggestion
    public static final String WALLET_NAME = "test wallet name";
```
?

**Review by dawciobiel — DISMISSED** [review]
LGTM

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletCreateControllerTest.java (hunk 1 above)
Comment (RobertoJavaDev) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletCreateControllerTest.java]: Good idea - changed

**Review by dawciobiel — APPROVED** [review]
LGTM

**Pull request merged**